platform/mbed_wait_api.h

```diff
@@ -46,54 +46,14 @@ extern "C" {
  * int main() {
  *     while (1) {
  *         heartbeat = 1;
- *         wait(0.5);
+ *         wait_us(5000);
```

Comment:
Why not `thread_sleep_for`?

Reply:
This example code is to show usage of the wait API, so I have used wait_us API.

Reply:
Not this example.
If you want to demonstrate usage of `wait_us` then you need an example of wait from ISR context or example of spin waiting.

Reply:
or perhaps wait for less than 1ms